tox.ini

```diff
@@ -143,7 +144,7 @@ passenv =
 deps =
     PyYAML
     regendoc>=0.8.1
-    sphinx
+    sphinx>=8.1.3
```

Comment:
I wonder if this should instead just reference the requirements file or the base section value...